tests/unit/test_module_conversion.py

```diff
@@ -23,6 +23,10 @@ def test_instant_message_example():
     assert isinstance(instant_message_example.recipient.screen_name, basestring)
     assert isinstance(instant_message_example.message, basestring)
     assert isinstance(instant_message_example.metadata.latency, float)
+    assert isinstance(instant_message_example.metadata.inner.a, float)
+    assert isinstance(instant_message_example.metadata.inner.layer.client.name, basestring)
+    assert isinstance(instant_message_example.metadata.inner.layer.status, Number)
+    assert isinstance(instant_message_example.client, Number)
```

Comment:
👍 

Would you also mind adding just one more test? The following would be fine:
```python
def test_nested_strategies_produce_data():
    """Ensure nested messages are accessible within strategy dict."""
    protobuf_strategies = modules_to_strategies(im_pb2)
    assert protobuf_strategies[im_pb2.MetaData.Inner].example()
    assert protobuf_strategies[im_pb2.MetaData.Inner.LimboDreamLayer].example()
```

Reply:
Done :)